tests/python/test_address.py

```diff
@@ -28,6 +28,12 @@
 
 from smartredis import Client
 
+def test_serialization(use_cluster, context):
+    # get env var to set through client init
+    ssdb = os.environ["SSDB"]
+    c = Client(address=ssdb, cluster=use_cluster, logger_name=context)
+    print(c)
```

Comment:
Similar worry to dataset test: this test should pass reguardless of changes made in this PR. Maybe do the same `repr(c) != str(c)` trick?

Reply:
Done